contracts/XENFT.sol

```diff
@@ -32,9 +32,10 @@ contract XENFT is IXENTorrent, IXENProxying, ERC721("XENFT", "XENFT") {
     // original contract marking to distinguish from proxy copies
     address private immutable _original;
     // ever increasing counter for NFT tokenIds, also used as salt for proxies' spinning
-    // TODO: make the next 2 public ???
-    uint256 private _tokenIdCounter = LIMITED_SERIES_COUNT;
```

Comment:
I believe these vars should be renamed with the leading underscore removed since they would no longer be private.